models/threshold/Threshold.adoc

```diff
@@ -0,0 +1,83 @@
+= ThresholdHealth
+:toc: macro
+:toc-title:
+
+== Introduction
+
+Kiali provides a customize configuration for ThresholdAlert from requests in your environment.
```

Comment:
First comment about naming.

"ThresholdAlert" is not correct. 
The goal of this feature is to extract the health logic to Kiali configuration in a global way, so a user could change it.

But using terms like "alert", "rule", "threshold" creates confusion with an alerting engine, which is not the goal of this feature.

Perhaps it's better the think on this like "HealthConfig", about "rules", "threshold" I will continue reading the whole PR before to suggest different naming but I understand some of concerns from @jshaughn related naming as these wording is used in differente environments.